src/main/resources/assets/securitycraft/lang/ja_jp.json

```diff
@@ -578,8 +604,17 @@
 	"block.securitycraft.username_logger": "ユーザー名記録装置",
 	"commands.securitycraft.bug": "バグの報告はDiscordサーバーの#bugreportチャンネルでお願いします。最初にルールをお読みください。",
 	"commands.securitycraft.connect": "IRC チャットは、SecurityCraft の v1.8.3 でサポートされなくなりました。ヘルプが必要な場合は、SecurityCraft の Discord の #help チャンネルを参照してください。",
+	"commands.securitycraft.convert.fill.failed": "指定されたエリアに変換可能なブロックはありません",
+	"commands.securitycraft.convert.fill.success": "%s個のブロックの変換に成功しました",
+	"commands.securitycraft.convert.set.failed": "指定された位置に変換可能なブロックはありません",
+	"commands.securitycraft.convert.set.success": "%s, %s, %sのブロックを変換しました",
 	"commands.securitycraft.dump.notFound": "SecurityCraftでは\"%s\"に登録されているものはありません。",
 	"commands.securitycraft.dump.result": "ここをクリックすると、%sレジストリエントリをクリップボードにコピーできます。",
+	"commands.securitycraft.help": "%s: %s + %s",
```

Comment:
Unnecessary because it is unchanged from en_us

Reply:
Thank you for reply. Fixed these changes.